index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+    <h1>Melanie J. Morris</h1>
+    <img
+        src="/home/melanie/Desktop/Stack/Week2/Portfolio-Site-Skeleton/Stack/Week2/Portfolio-Site-Skeleton/Images/mjm.jpeg">
```

Comment:
This will only work on your local computer because the link is specific to your home directory. Please fix this so that when I open your work, I can see the images. 
You can do this by changing the src to "./Images/mjm.jpeg"
The "./" here says to look for a directory that is in the current project.
Please fix this, make another commit, and let me know when you are finished.